modules/oadp-installing-dpa-1-3.adoc

```diff
@@ -320,8 +321,7 @@ spec:
 <13> Specify a bucket as the backup storage location. If the bucket is not a dedicated bucket for Velero backups, you must specify a prefix.
 <14> Specify a prefix for Velero backups, for example, `velero`, if the bucket is used for multiple purposes.
 endif::[]
-
```

Comment:
Trivial, but you deleted an extra line here compared to `main`.

Reply:
Thanks. Changes made